iml-agent/Cargo.toml

```diff
@@ -41,6 +41,7 @@ url = "1.7"
 v_hist = "0.1.1"
 uuid = { version = "0.7", features = ["v4"] }
 parking_lot = "0.9"
+regex = "1.3.1"
```

Comment:
What are we using this for?

Reply:
The regex is used to get the time offset in the ntp daemon plugin.